test/unit/cli.test.js

```diff
@@ -27,8 +27,10 @@ describe('cli parser', function () {
         newmanCLI = require('../../bin/newman');
     });
 
-    it('should export a function', function () {
-        expect(cli).to.be.a('function');
+    // delete require cache to use program instance for consecutive runs.
+    beforeEach(function () {
```

Comment:
This looks like merge conflict. Update version is above at `L21`

Reply:
Fixed.